CONTRIBUTION.md

```diff
@@ -114,12 +114,18 @@ storiesOf('Molecules/Top content bar', module)
 ## How-to release a component
 
 - Ensure you have an npm account under the `@axa-ch` organisation
-- Create a new branch that follows this pattern: `release/<component-name>`
-- Add your component to `lerna.json`
+- Create a new _release branch_ that follows this pattern: `release/<component-name>`
+- Manually add your component to `lerna.json` and remove all not-to-be-released components from the same
+- commit and push your lerna.json changes to the release branch
 - Clean the main and components dependencies:
   - `npx lerna clean`
   - `rm -rf node_modules`
 - Run `npm install`
 - ~~Run `npm run release` or `npm run prerelease`~~
+- Ensure you are logged in to npm (`npm login`)
+- Ensure you have the necessary credentials to push to the Githup repo via commandline commands (the following lerna commands do this automatically)
 - Run `npm run test && npx lerna version && npx lerna publish from-package`
```

Comment:
Do we really need to clean the lerna json when using `npx lerna publish from-package`? Because I never use this command, I clear the lerna.json file, and then just do a `npx lerna publish`. Maybe clearing the lerna file is not necessary with the from-package option.

Reply:
This is the sequence of commands I copied from Marco. Feel free to experiment and optimize!